src/Uno.Extensions.Toolkit.UI/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Uno.Extensions.Toolkit
```

Comment:
```suggestion
﻿namespace Uno.Extensions
```
Sorry missed this before... but with this being on the HostBuilder to better match the other Extensions libraries HostBuilder extensions let's actually move this to the root Uno.Extensions namespace